m4/libevent_mbedtls.m4

```diff
@@ -18,20 +18,23 @@ case "$enable_mbedtls" in
                    [have_mbedtls=no],
                    [-lmbedtls -lmbedcrypto -lmbedx509 $EV_LIB_GDI $EV_LIB_WS32])
     LIBS="$save_LIBS"
-    test "$have_mbedtls" = "yes" && break
     esac
     CPPFLAGS_SAVE=$CPPFLAGS
     CPPFLAGS="$CPPFLAGS $MBEDTLS_INCS"
     AC_CHECK_HEADERS([mbedtls/ssl.h], [], [have_mbedtls=no])
     CPPFLAGS=$CPPFLAGS_SAVE
     AC_SUBST(MBEDTLS_INCS)
     AC_SUBST(MBEDTLS_LIBS)
-    case "$have_mbedtls" in
-     yes)  AC_DEFINE(HAVE_MBEDTLS, 1, [Define if the system has mbedtls]) ;;
-    esac
+    if test "$have_mbedtls" = "yes" ; then
+        AC_DEFINE(HAVE_MBEDTLS, 1, [Define if the system has mbedtls])
+    elif test "$have_mbedtls" = "yes" ; then
```

Comment:
I guess it should check that `enable_mbedtls` is `yes` ?

Reply:
Yes, it certainly should.  Now fixed.